lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -281,10 +292,13 @@ class Simulator {
   /**
    * Estimates the time taken to process all of the graph's nodes.
    * @param {Node} graph
+   * @param {{ignoreObserved?: boolean}=} options
    * @return {LH.Gatherer.Simulation.Result}
    */
-  simulate(graph) {
+  simulate(graph, options) {
+    options = Object.assign({ignoreObserved: false}, options);
     // initialize the necessary data containers
+    this._ignoreObserved = options.ignoreObserved;
```

Comment:
I'm really not sure what to call this thing, basically it's "don't try and stick just to how the page loaded in the trace, feel free to load it as efficiently as you think possible"

Reply:
yeah i don't love this term. :)  as you know i don't love reusing it in different places in diff contexts. 

is `reconstructTCPConnectionPooling` accurate?